src/partr.c

```diff
@@ -91,6 +91,16 @@ static inline void multiq_init(void)
     unbias_cong(heap_p, &cong_unbias);
 }
 
+// Returns number of tasks contained in the queues
+JL_DLLEXPORT uint64_t jl_task_queue_length(void)
+{
+    uint64_t total = 0;
+    for (int32_t i = 0; i < heap_p; ++i) {
+        total += jl_atomic_fetch_relaxed(&heaps[i].ntasks);
```

Comment:
The C standard says you cannot change one load without changing all of them to use `jl_atomic_fetch/store_relaxed`